nixos/modules/services/web-apps/mediawiki.nix

```diff
@@ -18,8 +18,7 @@ let
   cacheDir = "/var/cache/mediawiki";
   stateDir = "/var/lib/mediawiki";
 
-  # https://www.mediawiki.org/wiki/Compatibility
-  php = pkgs.php81;
+  php = cfg.phpPackage;
```

Comment:
We should probably replace all occurrences of PHP with cfg.phpPackage to avoid alias hell.

Reply:
Sure.